content/docs/user-guide/programming/components/_index.md

```diff
@@ -1,7 +1,7 @@
 ---
-description: Learn how to create and reflect components in C++ using the O3DE Component Entity System.
-linktitle: Component Development
+linkTitle: Component Development
 title: Programmer's Guide to Component Development
+description: Learn how to create and reflect components in C++ using the O3DE Component Entity System.
 weight: 200
```

Comment:
Let's make sure gem and component development appear near the top of the ToC, and in that order (to match their order of appearance in the gem and component reference sections).
```suggestion
weight: 140
```